openedx/features/subscriptions/api/v1/tests/test_models.py

```diff
@@ -0,0 +1,137 @@
+from datetime import date, datetime, timedelta
+import ddt
+import pytest
+
+from openedx.features.subscriptions.models import UserSubscription
+from openedx.features.subscriptions.api.v1.tests.factories import UserSubscriptionFactory
+from student.tests.factories import CourseEnrollmentFactory, UserFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+pytestmark = pytest.mark.django_db
```

Comment:
@rehan99000 
Can we remove pytest related stuff from python test files? Should not it be handled via configs? Did you try running tests without these markers?

Reply:
removed the marker